holoviews/plotting/bokeh/__init__.py

```diff
@@ -54,7 +54,7 @@
 
                 # Rasters
                 Image: RasterPlot,
-                GridImage: ImagePlot,
+                GridImage: RasterPlot,
```

Comment:
Unless I commit to releasing a geoviews version right away ``gv.Image`` inheritance would be broken by the function suggestion.